distribution/lib/Standard/Base/0.0.0-dev/src/Data/Number/Internal.enso

```diff
@@ -0,0 +1,1041 @@
+polyglot java import java.lang.Double
```

Comment:
Why is this file called `Internal`?

It feels weird that we have paths like `Standard.Base.Data.Number.Internal.Integer`. The `Integer` (and Number and Decimal) types are one of the most primitive types of the language, used everywhere. It feels weird that their name contains the `Internal` part which kind-of suggests that these are some internal things not meant for end-users. Can't we just move it to `Data/Number.enso` to make the path `Standard.Base.Data.Number.Internal.Integer`?

Reply:
Moved all of that to `Data/Numbers.enso`